t/api/10-jobgroups.t

```diff
@@ -130,18 +153,45 @@ subtest 'create parent group' => sub() {
     check_for_event(jobgroup_create => {group_name => 'Cool parent group'});
 };
 
+my $cool_group_id;
 subtest 'create job group' => sub() {
-    $t->post_ok(
-        '/api/v1/job_groups',
-        form => {
-            name                        => 'Cool group',
-            size_limit_gb               => 200,
-            description                 => 'Test2',
-            keep_important_logs_in_days => 45
-        });
-    my $new_id = $t->tx->res->json->{id};
+    for my $form (
+        {size_limit_gb                  => '-300GB'},
+        {build_version_sort             => 'true'},
+        {keep_logs_in_days              => '18 days'},
+        {keep_important_logs_in_days    => '4 days'},
+        {keep_results_in_days           => '30 days'},
+        {keep_important_results_in_days => '1 days'},
+        {default_priority               => 'inherit'},
```

Comment:
It would actually make sense to allow this so one can revert to the inherited value. But it looks like that is really not supported so far so I'm fine considering this invalid.

Reply:
Yes, and since we support that with job templates one could be forgiven for trying to use that. So I picked the `invalid` value here as well to highlight that it is not supported.